rbac/management/workspace/utils.py

```diff
@@ -106,3 +107,15 @@ def _get_group_list_from_resource_definitions(resource_definitions: dict) -> lis
         except (ValueError, TypeError):
             group_list.remove(gid)
     return group_list
+
+
+def check_total_workspace_count_exceeded(request) -> bool:
```

Comment:
There are 4 levels where we can put validations for workspaces:
- Model level (uses only the instance itself for validation)
- service level (business rules, for example checking workspace limits for an org before creation)
- serializer level (uses raw input data from API requests(good validation - field formats, required fields)
- view level (used for request-specific checks - permission checks,..)

So 
`check_total_workspace_count_exceeded` - this is  service level  (is it business rule - it is not a "technical" limitation )
need to go into,  
`./workspace/service.py`
as is for example `_enforce_hierarchy_depth`

let me know if you have questions!